model/database/spans.yaml

```
@@ -701,3 +704,11 @@ groups:
       - ref: server.port
         requirement_level:
           conditionally_required: If not default (443).
+      - ref: db.cosmosdb.consistency_level
+        requirement_level:
+          conditionally_required: If available.
+      - ref: db.cosmosdb.regions_contacted
+        note: >
+            When `db.cosmosdb.regions_contacted` attribute is populated, it would indicate than the operation was performed on multiple regions i.e. cross-regional call.
```

Is this statement true? I thought this was populated for single region calls too

```suggestion
            When the `db.cosmosdb.regions_contacted` attribute is populated, it indicates that the operation was performed on multiple regions i.e. cross-regional call.
```

---

yes for single region, there would be single value, and multiple regions, there would be multiple values. But this is the only attribute which indicates that, this operation call was cross regional.